bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -37,23 +38,35 @@ const TEMPLATE =
 class ProjectBuilder {
     constructor (rootDirectory) {
         this.root = rootDirectory || path.resolve(__dirname, '../../..');
-        this.binDir = path.join(this.root, 'app', 'build', 'outputs', 'apk');
+        this.apkDir = path.join(this.root, 'app', 'build', 'outputs', 'apk');
+        this.aabDir = path.join(this.root, 'app', 'build', 'outputs', 'bundle');
     }
 
     getArgs (cmd, opts) {
-        if (cmd === 'release') {
-            cmd = 'cdvBuildRelease';
-        } else if (cmd === 'debug') {
-            cmd = 'cdvBuildDebug';
-        }
+        var args;
+        if (opts.packageType === PackageType.BUNDLE) {
+            if (cmd === 'release') {
```

Comment:
It would be quite a nested ternary statement if we merge both the `packageType` comparison and the `cmd` comparison together. There are 4 different outcomes for `buildCmd`.

Therefore the ternary statement would look more like:
```javascript
const buildCmd =
    (opts.packageType === PackageType.BUNDLE)
        ? (cmd === 'release')
            ? ':app:bundleRelease'
            : ':app:bundleDebug'
         : (cmd === 'release')
            ? 'cdvBuildRelease'
            : 'cdvBuildDebug';
```
I don't have a problem making that change but personally I find that more confusing to look at.

Reply:
> Therefore the ternary statement would look more like:
> […]

Agreed on my part.

Leaving it up to you. Maybe better to leave some of these ideas for a followup PR, as these changes seem to have good test coverage.

Reply:
if/else is perfectly readable here IMO. a bit verbose, but simple and effective.

Reply:
Resolving this conversation. If #786 dictates style changes, then another PR can be submitted.